src/pages/Event/Event2023FallMissions.tsx

```diff
@@ -7,10 +11,22 @@ import WhiteContainer from "components/WhiteContainer";
 
 import theme from "tools/theme";
 
-const MissionContainer = () => {
+import { ReactComponent as CreditIcon } from "static/events/2023fallCredit.svg";
+
+type MissionContainerProps = {
+  quest?: Quest;
```

Comment:
`quest` optional 한 value가 아니라 필수 필드로 사용해도 될 것 같아요.
`quest` 가 없으면 `MissionContainer` 생성에 의미가 없습니다.

```suggestion
  quest: Quest;
```